### DISCUSSION TITLE
[Solved] Nzyme in Docket / Container

### DISCUSSION THREAD

**Discussion by a020088** [nzymedefense/nzyme#677] [Questions]
I am now using Nzyme but I am wondering whether anybody has already placed Nzyme in a container. I did it already successfully but I always failed for USB WLAN Stich forwarding into the container. 

Does anybody solved this already?
Thanks Andreas

**Comment by lennartkoopmann** [comment]
AFAIK you can't easily pass the WiFi interfaces to Docker. That's why I never attempted it.
